src/service/run-api-service.js

```diff
@@ -88,6 +88,12 @@ module.exports = function (config) {
         id: '',
 
         /**
+         * Flag determines if X-AutoRestore: true header is sent to Epicenter. Defaults to true.
+         * @type {boolean}
+         */
+        autoRestoreRun: true,
```

Comment:
let's just call this `autoRestore`; "run" is redundant since it's part of the run service anyway